data/code/workflows/trigger-with-branding-tenant.ts

```diff
@@ -79,14 +79,19 @@ $client->workflows()->trigger('reservation-reminder-email', [
 ctx := context.Background()
 knockClient, _ := knock.NewClient(knock.WithAccessToken("sk_12345"))
 
-result, _ := knockClient.Workflows.Trigger(ctx, &knock.TriggerWorkflowRequest{
+request := &knock.TriggerWorkflowRequest{
   Workflow:   "reservation-reminder-email",
   Data: map[string]interface{}{
       "reservation_id": reservation.ID
   },
-  Recipients: reservationGuestIds,
   Tenant: "black-lodge"
-  })
+}
+
+for _, r := range reservationGuestIds {
+  request.AddRecipientByID(r)
+}
+
+result, _ := knockClient.Workflows.Trigger(ctx, req, nil)
```

Comment:
```suggestion
result, _ := knockClient.Workflows.Trigger(ctx, request, nil)
```
I think either this or we should refactor `request` above to match other examples? Also realizing I'm late to the party but still noting here so we don't forget

Reply:
good 👁️, I'll open a followup PR in which I'll update all exampls to use `request` rather than `req` to fix this and be consistent everywhere else